language/testing-infra/transactional-test-runner/src/framework.rs

```diff
@@ -192,8 +192,8 @@ pub trait MoveTestAdapter<'a> {
                             data_path.to_owned(),
                         )?;
                         match unit {
-                        CompiledUnit::Script { script, .. }=> (script, warning_opt),
-                        CompiledUnit::Module { .. } => panic!(
+                        AnnotatedCompiledUnit::Script(annot_script) => (annot_script.script.script, warning_opt),
```

Comment:
`.module.module` and `.script.script` is amusing :) might have been helpful to have a access function there, but it's fine

Reply:
I might change these to `.named_module.module` and `.named_script.script` since that seems more descriptive and less weird :)